src/Sylius/Bundle/CoreBundle/Doctrine/ORM/QueryBuilder/QueryBuilderPagination.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Doctrine\ORM\QueryBuilder;
+
+use Doctrine\ORM\QueryBuilder;
+
+class QueryBuilderPagination
```

Comment:
I do not think this class extraction is needed right now - it's used only in one place, so it seems to do more harm than good 🖖